org.o20.word.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "org.o20.word",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "word64.run",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=host" ],
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "o20coreapps",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                     "type": "git",
```

Comment:
can you fix the indentig here at the same time you pin the release to a specific version.